src/lib/markbind/src/parser.js

```diff
@@ -166,12 +166,39 @@ Parser.prototype.getMissingIncludeSrc = function () {
   return _.clone(this.missingIncludeSrc);
 };
 
+Parser.prototype._preprocessThumbnails = function (node) {
+  const element = node;
+  const isImage = _.hasIn(element.attribs, 'src') && element.attribs.src !== '';
+  if (isImage) {
+    return element;
+  }
+  const text = _.hasIn(element.attribs, 'text') ? element.attribs.text : '';
+  if (text === '') {
+    return element;
+  }
+  const renderedText = md.render(text);
+  const chr = cheerio.parseHTML(renderedText);
+  chr.map((child) => {
+    if (child.name === 'p') {
```

Comment:
You should add a comment on why changing from `p` to `span` is necessary.

Reply:
changed to use `md.renderInline`, no longer need to change `p` to `span`